The plugin no longer builds and runs with Java 8

The jar `spdx-rdf-store` seems to be compiled to Java 11?

This POM says Java 8 but running `mvn clean install` for this plugin says:
```
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] /C:/Users/ggregory/git/g/spdx-maven-plugin/src/main/java/org/spdx/maven/SpdxDependencyInformation.java:[69,29] cannot access org.spdx.spdxRdfStore.RdfStore
  bad class file: C:\Users\ggregory\.m2\repository\org\spdx\spdx-rdf-store\1.1.3\spdx-rdf-store-1.1.3.jar(org/spdx/spdxRdfStore/RdfStore.class)
    class file has wrong version 55.0, should be 52.0
    Please remove or make sure it appears in the correct subdirectory of the classpath.
[INFO] 1 error
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  6.531 s
[INFO] Finished at: 2023-02-04T10:18:15-05:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project spdx-maven-plugin: Compilation failure
[ERROR] /C:/Users/ggregory/git/g/spdx-maven-plugin/src/main/java/org/spdx/maven/SpdxDependencyInformation.java:[69,29] cannot access org.spdx.spdxRdfStore.RdfStore
[ERROR]   bad class file: C:\Users\ggregory\.m2\repository\org\spdx\spdx-rdf-store\1.1.3\spdx-rdf-store-1.1.3.jar(org/spdx/spdxRdfStore/RdfStore.class)
[ERROR]     class file has wrong version 55.0, should be 52.0
[ERROR]     Please remove or make sure it appears in the correct subdirectory of the classpath.
[ERROR]
[ERROR] -> [Help 1]
```

Ah... the problems seems to be an eager update to Apache Jena, building `spdx-rdf-store` with Java 8 yields:
```
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] /C:/Users/ggregory/git/g/spdx-java-rdf-store/src/main/java/org/spdx/spdxRdfStore/RdfStore.java:[38,29] cannot access org.apache.jena.graph.Node
  bad class file: C:\Users\ggregory\.m2\repository\org\apache\jena\jena-core\4.7.0\jena-core-4.7.0.jar(org/apache/jena/graph/Node.class)
    class file has wrong version 55.0, should be 52.0
    Please remove or make sure it appears in the correct subdirectory of the classpath.
```
Please go back to supporting Java 8 or make it obvious that Java 11 is required.

--- a/pom.xml
+++ b/pom.xml
@@ -17,8 +17,7 @@
   </organization>
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <maven.compiler.source>11</maven.compiler.source>
-    <maven.compiler.target>11</maven.compiler.target>
+    <maven.compiler.release>11</maven.compiler.release>
     <maven.version>3.8.5</maven.version>
     <sonar.host.url>https://sonarcloud.io</sonar.host.url>
     <sonar.organization>spdx</sonar.organization>
@@ -146,12 +145,17 @@
        </resource>
     </resources>
     <plugins>
-	<plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+		<artifactId>maven-compiler-plugin</artifactId>
+		<version>3.6.1</version>
+      </plugin>
+	  <plugin>
 		<groupId>org.owasp</groupId>
 		<artifactId>dependency-check-maven</artifactId>
 		<version>${dependency-check-maven.version}</version>
-	</plugin>
-    <plugin>
+	  </plugin>
+      <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-plugin-plugin</artifactId>
         <version>3.6.0</version>
@@ -174,20 +178,20 @@
           </execution>
         </executions>
       </plugin>
-        <plugin>
-          <groupId>org.apache.maven.plugins</groupId>
-          <artifactId>maven-source-plugin</artifactId>
-          <version>2.2.1</version>
-          <executions>
-            <execution>
-              <id>attach-sources</id>
-               <phase>verify</phase>
-               <goals>
-                 <goal>jar-no-fork</goal>
-               </goals>
-             </execution>
-           </executions>
-        </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-source-plugin</artifactId>
+        <version>2.2.1</version>
+        <executions>
+          <execution>
+            <id>attach-sources</id>
+             <phase>verify</phase>
+             <goals>
+               <goal>jar-no-fork</goal>
+             </goals>
+           </execution>
+         </executions>
+      </plugin>
     </plugins>
   </build>
   <profiles>

--- a/pom.xml
+++ b/pom.xml
@@ -17,8 +17,8 @@
   </organization>
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <maven.compiler.source>1.8</maven.compiler.source>
-    <maven.compiler.target>1.8</maven.compiler.target>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
     <maven.version>3.8.5</maven.version>
     <sonar.host.url>https://sonarcloud.io</sonar.host.url>
     <sonar.organization>spdx</sonar.organization>

